### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by smcclellan**
This includes some basic testing of the functionality that we expect users of
the gem to require. It is by no means exhaustive, but just a start to get
things rolling.

For https://tickets.puppetlabs.com/browse/PE-14831

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+  require 'rake'
+  GettextSetup.initialize(File.absolute_path(File.join('spec', 'fixtures', 'locales'), File.dirname(__FILE__)))
+  Dir.chdir('spec/fixtures')
+  Rake.application['gettext:pot'].invoke
+  # No use in running these without Transifex integration to actually translate
+  # strings.
+  # Rake.application['gettext:po'].invoke('de')
+end
+
+# if defined?(RSpec::Core::RakeTask)
```
Comment (libbymo) [inline Rakefile]: Should these come out?

**Comment by libbymo** [comment]
Other than the one nit about the comments, this makes me happy! :)

Thanks, @smcclellan !

**Comment by smcclellan** [comment]
@libbymo I have removed the comment on that line.

**Comment by steveax** [comment]
👍

**Pull request merged**